grove/web/util.go

```diff
@@ -198,3 +200,145 @@ func ParseHTTPDate(d string) (time.Time, bool) {
 
 // RemapTextKey is the plugin shared data key inserted by grovetccfg for the Remap Line of the Delivery Service in Traffic Control, Traffic Ops.
 const RemapTextKey = "remap_text"
+
+const Day = time.Hour * time.Duration(24)
+
+// GetHTTPDeltaSeconds is a helper function which gets an HTTP Delta Seconds from the given map (which is typically a `http.Header` or `CacheControl`. Returns false if the given key doesn't exist in the map, or if the value isn't a valid Delta Seconds per RFC2616§3.3.2.
+func GetHTTPDeltaSeconds(m map[string][]string, key string) (time.Duration, bool) {
```

Comment:
Is there a big reason these rules functions were moved? This function, and all the helper functions (`getCurrentAge`, `correctedInitialAge`, etc) are all very specifically implementing RFC 7234 and RFC 2616 rules. 

The `rules.go` file is intended to encapsulate those RFC rules, so someone doesn't think they can just change them to do what they need at the time. To make it more obvious "this file and these functions specifically implement the RFC". So, I'd kind of like to keep the Cache-Control RFC logic there.

I've considered even giving `rules.go` its own package, for that reason.

Reply:
I needed it outside the package, and it was lowercase in rules.go. I read rules as in remap-rules, not rfc-rules... So I thought web would be a better package, because I thought they're more general than remap-rules... +1 on making it's own package, want me to give that a try? What would you like to call it?

Reply:
How about `rfc`? Can you add a Godoc package-level comment too, along the lines of

```
// Package rfc contains functions implementing RFC 7234, 2616, and other RFCs
// When changing functions, be sure they still conform to the corresponding RFC.
// When adding symbols, document the RFC and section they correspond to.
package rfc
```